modules/installation-creating-aws-vpc-carrier-gw.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones)
+// * post_installation_configuration/aws-compute-edge-tasks-zones.adoc (AWS zone tasks) 
+
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
+ifeval::["{context}" == "aws-compute-edge-tasks-zones"]
+:post-aws-zones:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-creating-aws-vpc-carrier-gw_{context}"]
+ifdef::wavelength-zone[]
+= Creating a VPC carrier gateway
+endif::wavelength-zone[]
+ifdef::post-aws-zones[]
+= Wavelength Zones only: Creating a VPC carrier gateway
+endif::post-aws-zones[]
+
+To use public subnets, such as deploying load balancers or edge worker nodes, in your {product-title} cluster on {zone-types}, you must create the carrier gateway and associate it to the VPC. Carrier gateways exist for VPCs that only contain subnets in a Wavelength Zone. A carrier gateway supports IPv4 traffic.
+
+The following list explains functions of a carrier gateway in the context of an AWS {zone-type} location: 
+
+* Provides connectivity between your Wavelength Zone and the carrier network, which includes any available devices from the carrier network.
+* Performs Network Address Translation (NAT) functions, such as translating IP addresses, which are public IP addresses stored in a network border group, from {zone-type} to carrier IP addresses. These translation functions apply to inbound and outbound traffic.
```

Comment:
`from Local Zones or Wavelength Zones to carrier IP addresses` presents in the doc, should be `from Wavelength Zones to carrier IP addresses`?